.eslintrc

```diff
@@ -3,10 +3,24 @@
   "root": true,
   "overrides": [
     {
-      "files": ["packages/grafana-ui/**/*/!(*.story).{ts,tsx}", "public/app/**/*.{ts,tsx}"],
+      "files": [
+        "packages/grafana-ui/**/*/!(*.story).{ts,tsx}",
+        "packages/jaeger-ui-components/**/*.{ts,tsx,js}",
+        "public/app/**/*.{ts,tsx}"
+      ],
       "rules": {
         "react-hooks/rules-of-hooks": "off",
-        "react-hooks/exhaustive-deps": "off"
+        "react-hooks/exhaustive-deps": "off",
+        "react/prop-types": "off",
+        "react/no-children-prop": "off",
+        "react/no-unescaped-entities": "off",
+        "react/jsx-no-target-blank": "off",
+        "react/display-name": "off",
+        "react/jsx-key": "off",
+        "react/no-deprecated": "off",
+        "react/no-unknown-property": "off",
+        "react/no-find-dom-node": "off",
+        "react/no-render-return-value": "off"
```

Comment:
This rules to be gradually enabled when we kick in the updated grafana-esint-config